### PR TITLE
add a SECURITY notice 

### PR DESCRIPTION
**Pull request by mymphe**
This PR adds a SECURITY.md file which refers users to Immunefi

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+The Lido team appreciates your efforts to responsibly disclose your findings and will make every effort to acknowledge your contributions.
```
Comment (TheDZhon) [inline SECURITY.md:5]: No 'Lido team' please :)

Lido contributors :pray:

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Love it!

Though please tune up wording a bit

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+To report a security issue, please use the [Lido Bug Bounty program on Immunefi](https://immunefi.com/bounty/lido).
+
+The Immunefi platform allows the Lido team to better track and respond to vulnerabilities, while also providing an opportunity for researchers to be rewarded for their findings.
+
+Please DO NOT file a public issue on GitHub or disclose the vulnerability publicly in any way before it has been addressed by the Lido core contributors.
```
Comment (tamtamchik) [inline SECURITY.md:11]:
```suggestion
> [!IMPORTANT]
> Please DO NOT file a public issue on GitHub or disclose the vulnerability publicly in any way before it has been addressed by the Lido core contributors.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Scope
+
+For details on what is considered in scope for the bug bounty program, please refer to the Lido Bug Bounty page on Immunefi.
+
+## Process
+
+1. Submit your report through the Immunefi platform.
+2. The Lido team will assess the report and may ask for additional information or clarification.
```
Comment (tamtamchik) [inline SECURITY.md:20]:
```suggestion
2. The Lido contributors will evaluate your report and may request additional information or clarification.
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+3. Once validated, the team will work on a fix and coordinate the release process.
+4. After the fix has been deployed, the team will publish a security advisory and acknowledge your contribution (if you wish to be credited).
```
Comment (tamtamchik) [inline SECURITY.md:22]:
```suggestion
4. After the fix is deployed, a security advisory will be published, and your contribution will be acknowledged (if you wish to be credited).
```

**Review by TheDZhon — APPROVED** [review]
LGTM

**Pull request merged**